### DISCUSSION TITLE
I just can't make it work

### DISCUSSION THREAD

**Discussion by Alp136** [bangcorrupt/hacktribe#270] [Q&A]
Hi ! sorry for the help I ask for, it may not be your first time.

So english is not my first language, and I just absolutely don't understand what i'm doing.
I've tried to install the hacktribe editor, and the hacktribe-gui.exe, when I follow the instructions, it says "File not found: hacktribe\hash\SYSTEM.VSB.sha
Source hash file not found"

I've already downloaded the vsb from korg website, the electribe sampler 2.02 one, I don't know what I'm doing wrong.

I already tried to follow the special instruction you gave in the Q&A, but I don't understand anything, I also tried to install the dependancies, but I'm lost with all the files to download, and I don't know what to do with them..

Thanks for the help !

**Comment by bangcorrupt** [comment]
Answered in #272.
